docs/RC/SetGlobalProperties/index.md

```diff
@@ -0,0 +1,97 @@
+## SetGlobalProperties
+
+Type
+: Function
+
+Sender
+: SDL
+
+Purpose
+: Inform HMI of user location properties set by the application for RC modules allocation purposes
+
+### Description
+SDL requests to set the values of user location sent by RC application. The default `userLocation` should be `DRIVER`.  
+In case driver's seat is out of the service area of a module, driver's application should be able to control any free module that is allowed by policies.
+
+### Request
+#### Behavior
+
+!!! MUST   
+
+1. Store the userLocation and associate it with appID.
+2. Respond to the request.
```

Comment:
Probably not necessary to state that the HMI must respond to the RPC request.

Reply:
removed in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/171/commits/de249c04759674b8befb3be7d380a6d5e27820b9